SIT.Manager.Avalonia/Models/LocalizationConfig.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Avalonia.Markup.Xaml.Styling;
+using System.Linq;
+using System;
+
+namespace SIT.Manager.Avalonia.Models
+{
+    public static class LocalizationConfig
+    {
+        public enum Languages
```

Comment:
We should probably use CultureInfo over our own enums

Reply:
> We should probably use CultureInfo over our own enums

Problem behind Culture Info it's not 100% correct when you are starting the application. But still a good point to set at the very first start of the application the language based on what language you have on computer.